citadel/indico_citadel/plugin.py

```diff
@@ -75,13 +82,15 @@ class CitadelPlugin(LiveSyncPluginBase):
         'num_threads_files': 5,
         'num_threads_files_initial': 25,
         'disable_search': False,
+        'large_category_warning_threshold': 0,
     }
     backend_classes = {'citadel': LiveSyncCitadelBackend}
 
     def init(self):
         super().init()
         self.connect(signals.core.get_search_providers, self.get_search_providers)
         self.connect(signals.plugin.cli, self._extend_indico_cli)
+        self.template_hook('event-category-threshold-alert', check_event_categories)
```

Comment:
This name is way too specific - template hooks never describe a particular thing they are used for but something generic, like `category-protection-page`